docs: account enumeration mitigation (#2126)
ory/docs: docs/identities/sign-in/identifier-first-authentication.mdx

Identifier first authentication first requests the user's identifier such as an email or username before prompting for a password
or other authentication methods.

:::note

Identifier first authentication is required when using B2B Organization login.

> :::

Reviewer: This note should be on the orgs docs, I think.

Author: I think in both, no?

Author: Well, ideally this would be in the product itself and does not require the user to know what knobs to turn!

Author: added the note

```

To disable this feature, set `style` to `unified`.

## Account enumeration mitigation

Account enumeration mitigation prevents malicious actors from being able to identify if a user exists or not.

> Per default, account enumeration mitigation is disabled when using identifier first authentication. This improves user experience
> as the user quickly sees if they are signed up or not with the chosen identifier (email / username). To enable account
> enumeration, use the Ory CLI patch command

Reviewer: ```suggestion
By default, Ory does not prevent account enumeration in the identifier first authentication flow. This improves user experience
as the user quickly knows if they have an account with the chosen identifier (email / username) or not. To enable account
enumeration, use the Ory CLI patch command
```
